docs/hub/webhooks.md

```diff
@@ -0,0 +1,120 @@
+# Webhooks
+
+Webhooks allow you to know when new changes happen on specific repos or repos belonging to specific users / organizations (not just your repos, but any repo!).
+
+You can configure the webhooks in your [settings](https://huggingface.co/settings/webhooks).
+
+You can watch repo changes or community events. If you watch all repos belonging to a user or an organization, you can also receive events for repo creation.
+
+## Webhook payloads
+
+After registering a webhook, you will be notified of events via an `HTTP POST` call on the specified URL. The payload is encoded in JSON.
+
+To determine the nature of the event, you can look at the `event` and `domain` fields in the payload.
+
+You can check the last payloads sent in the activity tab of the webhook page, as well as replay webhooks.
+
+### Event & Domain
+
+The top-level properties `event` and `domain` are always specified and used to determine the nature of the event.
+
+`event` can take one of the following values: 
+
+- `"create"`
+- `"delete"`
+- `"update"`
+- `"move"`
+
+`domain` specifies the domain of the event. It can be one of the following values:
+
+
+- `["repo"]` - Global events on repos: create, delete, move
+- `["repo", "content"]` - Events on the content of the repo, like new commits or tags. Triggers on new pull requests as well due to the newly created reference / commit.
+- `["repo", "config"]` - Events on the config: update space secrets, update settings, update DOIs, disabled or not, ...
+- `["community", "discussion"]` - Creating a discussion or pull request, updating the title or status
+- `["community", "discussion", "comment"]` - Creating, updating, hiding a comment
+
+⚠️ **You should consider that any event on a more granular domain is an `"update"` on the encompassing domains.** ⚠️
+
+For example, any event on `["community", "discussion", "..."]` is an `"update"` on `["community", "discussion"]`.
+
+This is important in case more granularity is added to the domains by Hugging Face. For example if the domain domain `["repo", "config", "doi"]` is added, it would be accompanied by an `"add"` event when DOIs are created. 
```

Comment:
```suggestion
This is important in case more granularity is added to the domains by Hugging Face. For example if the domain `["repo", "config", "doi"]` is added, it would be accompanied by an `"add"` event when DOIs are created. 
```

Don't understand this either